frontend/src/metabase-types/api/util.ts

```diff
@@ -41,3 +41,5 @@ export interface BugReportDetails {
   "metabase-info": MetabaseInfo;
   "system-info": SystemInfo;
 }
+
+export type LongTaskStatus = "incomplete" | "complete" | "aborted";
```

Comment:
Don't think there's a point to refactoring this into it's own constant if we only use it for `InitialSyncStatus`, and don't re-use it elsewhere. The `Download` interface in `frontend/src/metabase-types/store/downloads.ts` uses a different type (`status: "complete" | "in-progress" | "error"`).

Reply:
This is a store state but the components that display the status use `"incomplete" | "complete" | "aborted"` which is coming from the BE for database entities. So to reuse the same components but with more correct semantic I added `LongTaskStatus` as a base type since `InitialSyncStatus` is not a very suitable name for downloads